api/agentpb/agent.proto

```diff
@@ -15,8 +15,9 @@ enum Type {
   MYSQLD_EXPORTER = 2;
   MONGODB_EXPORTER = 3;
   QAN_MYSQL_PERFSCHEMA_AGENT = 4;
-  QAN_MYSQL_SLOWLOG_AGENT = 6;
   POSTGRES_EXPORTER = 5;
+  QAN_MYSQL_SLOWLOG_AGENT = 6;
+  QAN_MONGO_PROFILER_AGENT = 7;
```

Comment:
Let's call it `qan-mongodb-profiler`, `qan_mongodb_profiler`, `QAN_MONGODB_PROFILER_AGENT`, `QANMongoDBProfilerAgent`, etc for consistency with Agent and Service types.